src/devtools/client/debugger/src/reducers/sources.ts

```diff
@@ -256,11 +250,31 @@ function update(state = initialSourcesState(), action: AnyAction) {
 
     case "SOURCES_LOADED":
       return { ...state, sourcesLoading: false };
+
+    case "SET_SELECTED_SOURCE_SCROLL_POSITION":
+      return {
+        ...state,
+        sourceScrollPositions: {
+          ...state.sourceScrollPositions,
+          [action.sourceId]: action.scrollPosition,
```

Comment:
:speech_balloon: yeah, really can't say I'm a fan of keeping scroll position in Redux, but 🤷‍♂️

Reply:
Cool beans. Let's use it instead then :)